content/cli/maint_cli/certificates.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: bt_wiki
+title: certificates
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+aliases: /cli/certificates/
+---
+
+{{%children style="h3" description="true"%}}
+
+The `cfy certificates` commands are used to handle procedures related to certificates. 
+
+## Commands
+
+### Replacing Certificates
+**Use case:** Whe the certificates on the cluster instances or on an AIO manager are about to expire and need to be replaced.  
```

Comment:
typo? probably meant "When"

I would consider something like:
Certificates are used by Cloudify for TLS based secure communication between the different Cloudify components and between the user interface and the Cloudify manager.
Certificates are set during the initial deployment of the Cloudify management (cluster or all-in-one), but later maintenance and replacement of the certificates may be required as a result of regulatory compliance demand, certificate expiration, or revocation due to security breach.
Follow this procedure when certificate replacement is required.